### ISSUE TITLE
Clear up the Documentation, Make it Clearer, with better examples.

### ISSUE THREAD

**Issue by AsrielDewitt** [hydrusnetwork/hydrus#1618]
I am a new user to this and I'm trying to just get some images from Twitter.

So obviously I went to the documentation an-HOLY FUCK

IT's bad, BAD BAD. The major sin of "Assume everybody already know" syndrome bad.

For example. to get the downloader, you click, it NEVER tells you where to find it on the program itself.

like Top Bar > Pages > Downloader> and choose.

Then there's the plugins, I look and I supposed to use NItter to download stuff from twitter. Okay, confusing way of importing stuff with images, done deal. Oh wait, NOthing in the documentation tells where what to do next after import the plugin, DO I use URL downloader? Do I use Simple Downloader? Watcher? NOTHING.

TLDR: The documentation is basically "Fuck all instructions, let me tell you what it does instead of how to use it."

**Comment by floogulinc** [comment]
Twitter is an especially bad site. If you want to get individual tweets the built in parsers in hydrus should work but for timelines you will either need to use the limited parser I posted on the discord or an external tool like hydownloader. You can thank Elon for that.

Regarding the documentation in general, we are aware it's not the best but also hydrus is a power user oriented program where you're expected to figure stuff out. If you're stuck though you can definitely ask on the Discord. 

Also, please feel free to make a pull request with docs improvements.

**Comment by Zweibach** [comment]
Pages is found under pages. This is not arcane knowledge or rocket science requiring six years in university. You are expected to actually use the program without us having to hold your hand every step of the way.

Or in your words: I opened the issue and HOLY FUCK!
It's bad, BAD BAD. It says nothing of use and expects people to know what to "fix".

Shit issue, closing.
